examples/flows/standard/conditional-flow-for-if-else/README.md

```diff
@@ -2,18 +2,18 @@
 
 This example is a conditional flow for if-else scenario.
 
+By following this example, you will learn how to create a conditional flow using the `activate config`.
```

Comment:
we can link back to the doc here

Reply:
The feature dco is not ready yet and I will update here once it is merged into the main.